### PR TITLE
Doc: Use proper name capitalization of tooling (Ruff, Black, Flake8)

### PR DESCRIPTION
**Pull request by Avasam**
A small documentation nitpick I had. Now using proper name capitalization of tooling (Ruff, Black and Flake8), which wasn't done consistently. Unless of course it's used in a command.
Also added some anchors.
If you disagree with any of those, lmk, I can update.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -68,33 +68,33 @@ exclude = [
...
+    "PYI025", # always alias `collections.abc.Set` as `AbstractSet` when importing it
+    "PYI032", # use `object`, not `Any`, as the second parameter to `__eq__`
+    "UP004", # Remove explicit `object` inheritance
+    "UP006", # PEP-585 autofixes
+    "UP007", # PEP-604 autofixes
+    "UP013", # Class-based syntax for TypedDicts
+    "UP014", # Class-based syntax for NamedTuples
+    "UP019", # Use str over typing.Text
+    "UP035", # import from typing, not typing_extensions, wherever possible
+    "UP039", # don't use parens after a class definition with no bases
```
Comment (srittau) [inline pyproject.toml:97]: Are these changes intentional?

**Review by srittau — APPROVED** [review]
LGTM, thanks!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -84,16 +87,18 @@ terminal to install all non-pytype requirements:
...
+also performed by [`Ruff`](https://github.com/astral-sh/ruff).
 
 The repository is equipped with a [`pre-commit.ci`](https://pre-commit.ci/)
 configuration file. This means that you don't *need* to do anything yourself to
 run the code formatters. When you push a commit, a bot will run those for you
 right away and add a commit to your PR.
 
 That being said, if you *want* to run the checks locally when you commit,
-you're free to do so. Either run `isort`, `black` and `ruff` manually...
+you're free to do so. Run either of the following manually...
```
Comment (AlexWaygood) [inline CONTRIBUTING.md:101]: Hmm, this changes the meaning of the sentence. Maybe this?

```suggestion
you're free to do so. Either run the following manually...
```

**Review by AlexWaygood — APPROVED** [review]
